### PR TITLE
D8CORE-1632: Add variables to quote template

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW 

# Summary
- Adds new variables to quote template to allow for different media options

# Review By (Date)
- May 1st?

# Urgency
- Low

# Steps to Test

1. Review code
2. Review render in build url

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- D8CORE-1632
- SOE Person type

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by jenbreese** [comment]
@sherakama can you restart the tugboat? Or are we totally over the quota?

File: core/src/templates/components/quote/quote.twig [hunk 1]
```diff
@@ -15,9 +16,12 @@
  */
 #}
 <figure {{ attributes }} class="su-quote {{ modifier_class }}">
-  {%- if src is not empty -%}
-  <img class="su-quote__img" src="{{ src }}" alt="{{ alt }}">
-  {%- endif -%}
+  {%- block block_quote_media -%}
```
Comment (yvonnetangsu) [inline core/src/templates/components/quote/quote.twig:19]: Actually could this use the nested media.twig template, like what we did for the Card component?

Comment (sherakama) [reply core/src/templates/components/quote/quote.twig:19]: Oh, that might be an option. Let me see about that refactor.

**Comment by sherakama** [comment]
> Just that question about nested media.twig, and if not, do we need any stying for {{ quote_media }}, eg, float:left?

Naw, I'd like this to be an open field, and developers can bring their own styles if they add something wacky to it.

**Comment by yvonnetangsu** [comment]
> > Just that question about nested media.twig, and if not, do we need any stying for {{ quote_media }}, eg, float:left?
> 
> Naw, I'd like this to be an open field, and developers can bring their own styles if they add something wacky to it.

@sherakama Please feel free to merge this in first so you can use the open {{ quote_media }} field. We can always decide whether we want the nested media.twig later.

**Comment by sherakama** [comment]
> @sherakama Please feel free to merge this in first so you can use the open {{ quote_media }} field. We can always decide whether we want the nested media.twig later.

Thanks, It doesn't appear I will be getting the cycles to get to that work so merging this as is because it doesn't hurt.

**Pull request merged**